eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/props/AppPropertiesLinks.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.emory.cci.aiw.cvrg.eureka.common.props;
+
+
+/**
+ *
+ * @author Miao Ai
+ */
+public class AppPropertiesLinks {
```

Comment:
Since this is a data transfer object, put it in edu.emory.cci.aiw.cvrg.eureka.common.comm.